app/assets/stylesheets/search.css.scss

```diff
@@ -1,46 +1,40 @@
-.page_result,
-.slack_result {
+.search_result {
+  display: block;
   margin-bottom: 10px;
-  padding: 15px;
+  padding: 15px 15px 0;
   border: solid 2px #eee;
   border-bottom: solid 3px #ddd;
   border-radius: 5px;
   color: #444;
-}
-
-.page_result {
-  .info {
-    margin-bottom: 5px;
-    color: #111;
-  }
 
-  .title {
-    font-size: 110%;
+  .label {
+    display: inline;
+    border-radius: 3px;
     font-weight: bold;
-  }
+    padding: 3px 5px;
 
-  .highlight {
-    background-color: #ff9;
-  }
-}
+    &.label-page {
+      background-color: #3cf;
+      border: solid 1px darken(#3cf, 20%);
+    }
 
-.slack_result {
-  .info {
-    margin-bottom: 5px;
-    color: #111;
-    font-size: 90%;
+    &.label-blog {
+      background-color: #f93;
+      border: solid 1px darken(#f93, 20%);
```

Comment:
Color literals like `#f93` should only be used in variable declarations; they should be referred to via variable everywhere else.